css/main.css

```diff
@@ -145,68 +149,77 @@ kbd {
 	display: block;
 }
 
+.flexTopBar {
+	display: flex;
+	flex-direction: column;
+	width: 100%;
+	top: 0;
+	left: 0;
```

Comment:
I don't think we need a top or left since we aren't using a position style. This would be good if it was sticky or some other position, but in this case, not needed.

Reply:
fixed